common/database/database_update_manifest.cpp

```diff
@@ -5116,6 +5116,21 @@ ALTER TABLE `object` CHANGE COLUMN `unknown20` `incline` int(11) NOT NULL DEFAUL
 ALTER TABLE `keyring`
 ADD COLUMN `id` int UNSIGNED NOT NULL AUTO_INCREMENT FIRST,
 ADD PRIMARY KEY (`id`);
+)"
+	},
+	ManifestEntry{
+		.version = 9247,
+		.description = "2023_12_14_starting_items_fix.sql",
+		.check = "SHOW COLUMNS FROM `starting_items` LIKE 'race_list'",
```

Comment:
We need something different to anchor on here that no longer be the case after we run the migration. Since we're mutating data we're not anchoring on any schema changes.

If I change my version back 30 versions and re-run everything, I should get all green. If I were to run updates again it will think I need to run this again and error out.

For example, if we also change a field in this migration from say `slot` to `inventory_slot` then we can anchor on something that truly changed schema wise in this migration. We would need to obviously update repository / code but it would make all of this seamless

Reply:
Can we check the order of the columns to confirm that it's correct?

Reply:
That's a good call out, we could do that

Reply:
Don't think `SHOW COLUMNS` tells us the ordinal position of the column, would we want to assume they have access to `information_schema` database and select its position from there?

Reply:
Yeah - not everyone gives their server user access to `information_schema` so it may be difficult